bootcompiler/src/main/scala/org/mozartoz/bootcompiler/transform/TreeCopier.scala

```diff
@@ -6,6 +6,16 @@ import ast._
 import symtab._
 
 class TreeCopier {
+  def copyOptions[U <: StatOrExpr](orig: Node, copy: U): U = {
```

Comment:
Yes, however, I was intending to use this mechanism for handling tail call marking as well, for consistency. What do you think about it?